coalib/core/Bear.py

```diff
@@ -130,6 +132,23 @@ class Bear:
     CAN_FIX = set()
     ASCIINEMA_URL = ''
     BEAR_DEPS = set()
+    """
+        Attributes for a new bear.
+
+        :param section:
+            The section object where bear settings are contained.
+        :param file_dict:
+            The file-dictionary containing a mapping of filenames to the
+            according file contents.
+        :raises RuntimeError:
+            Raised when bear requirements are not fulfilled.
```

Comment:
as we can't document `__init__` any more, we should move that to class level and embed it into the text somehow, like
```
On instantiation, a `RuntimeError` is raised when the bear requirements are not fulfilled.
```